torch/_torch_docs.py

```diff
@@ -1500,6 +1500,58 @@ def merge_dicts(*dicts):
     False
 """)
 
+add_docstr(torch.cov,
+           r"""
+cov(input1, input2=None, rowvar=True, bias=None, ddof=None, fweights=None, aweights=None) -> Tensor
+
+Estimates a covariance matrix, given input tensor data and weights (optional).
```

Comment:
This is OK but could you elaborate on how :attr:\`input\` relates to the estimated covariance matrix? That is, what is the conceptual structure of :attr:\`input\` (which is described in the "Args" section), and what is this estimating the covariance of? 

"Estimating" the covariance is also vague. Can this clarify how the estimate is computed, possibly with a mathematical expression?